src/azure-cli/azure/cli/command_modules/appconfig/keyvalue.py

```diff
@@ -171,6 +174,13 @@ def import_config(cmd,
     # append all feature flags to src_kvs list
     src_kvs.extend(src_features)
 
+    # In strict mode, delete kvs with specific label that are missing from the imported file
+    if strict:
+        kvs_to_delete = list(filterfalse(lambda kv: any(kv_import.key == kv.key and label == kv.label
+                                                        for kv_import in src_kvs), dest_kvs))
```

Comment:
>dest_kvs [](http://example.com/codeflow?start=83&length=8)

If user wants preview, we call `__discard_features_from_retrieved_kv(dest_kvs)`, which removes all features from `dest_kvs`. So feature flags will not be deleted in strict mode.

Reply:
Good catch!
fixed now.